SSH/run-tests.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```

Comment:
I'd add `set -e` here, if the script works with that? Otherwise e.g. error from `mktemp -d` may go undetected and result in wrong/insecure behavior.